README.md

```diff
@@ -98,27 +102,28 @@ We don't advise you to use the same EFS instance for persisting the data from mu
 
 ### 2. Add the Alfresco Helm repository to helm
 
-```helm repo add alfresco-incubator https://kubernetes-charts.alfresco.com/incubator```
+```
+helm repo add alfresco-incubator https://kubernetes-charts.alfresco.com/incubator
+helm repo add alfresco-stable https://kubernetes-charts.alfresco.com/stable
+```
+
+### 3. Configure domain in your values file
 
-### 3. Define a variable for your Route53 entry that you will use for the deployment
+Pull the values file from the current repo:
 
 ```bash
-export ELB_CNAME="YourDesiredCname.YourRoute53DnsZone"
-#example export ELB_CNAME="alfresco.example.com"
+curl -O https://raw.githubusercontent.com/Alfresco/alfresco-dbp-deployment/master/charts/incubator/alfresco-dbp/values.yaml
 ```
 
+Open it in your favorite text editor and replace all occurences of REPLACEME with your desired Route53 entry "YourDesiredCname.YourRoute53DnsZone"
```

Comment:
Devs are bound to make a mistake with this, and (as I think Kris mentioned) we've now strayed from an easy copy and paste of command lines (while changing a few env variables along the way).

Can we add an "... or run the following commands:" with an `export` setting the domain and a `sed` to replace in the values file?